src/pages/Profile.scss

```diff
@@ -0,0 +1,15 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700|Lora:400,700|Sacramento');
+
+.profile-container {
+  display: flex;
+  margin-top: 34px;
+  padding: 0% 6%;
```

Comment:
Any reason why we are using percentages instead of `rem` or `px`?

Reply:
Yes this padding is relative to the screen size, basically, it is a percentage of the width of the screen so it increases or decreases the padding by the percentage depending on the screen size